android/src/main/java/com/reactnativecommunity/webview/RNCWebViewManager.java

```diff
@@ -299,6 +299,21 @@ public void setHardwareAccelerationDisabled(WebView view, boolean disabled) {
     }
   }
 
+  @ReactProp(name = "androidLayerType")
+  public void setLayerType(WebView view, String layerTypeString) {
+    int layerType = View.LAYER_TYPE_NONE;
+    switch (layerTypeString) {
+        case "hardware":
+          layerType = view.isHardwareAccelerated() ? View.LAYER_TYPE_HARDWARE : View.LAYER_TYPE_SOFTWARE;
```

Comment:
Not sure if checking `view.isHardwareAccelerated() ` is necessary, but it feels safer. According to the [documentation](https://developer.android.com/reference/android/view/View#LAYER_TYPE_HARDWARE), when hardware acceleration is turned off, hardware layers behave exactly as software layers, but it's not clear for me how this would behave in case someone manually disables the HA and then sets the layer type to `hardware`